packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -54,15 +54,13 @@ The widget upgrade for a redirect sign-in flow depends on whether you configured
 
 To update the widget:
 
-> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
+1. In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
 
-1. In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
-
-2. Go to the **Settings** tab. (EA users: in the **Pages** tab, click **Configure** for the page that has the version you want to upgrade.)
+2. Go to the **Settings** tab. (If you enabled multibrand customization, in the **Pages** tab, click **Configure** for the page that has the version you want to upgrade.)
 
     - In the **Sign-In Widget Version** section, check that the **Version** is the highest version available. The widget is always the latest version if you're not using a custom URL domain.
     - If you're using the [custom domain feature](/docs/guides/custom-url-domain/) and the version isn't correct, you can pin the widget's version. Click **Edit** in the **Sign-In Widget Version** section and then select the **Version** field.
-    - (EA users: You can't customize the Sign-In Widget for the Okta default brand.)
+    - You can't customize the Sign-In Widget for the Okta default brand. See [Branding](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding).
```

Comment:
This needs to say, "If you enabled multibrand customization, You can't customize the Sign-In Widget for the Okta default brand".. The EA users wording I mean
In non-multibrand world, there'll only be one brand and it can be customized

Reply:
Fixed. Thank you :)